the-collab-lab/tcl-76-smart-shopping-list
# Installed MUI and display popup modal when shopping list is empty

## Description

This added an alert to the List page, for users without any items. The popup message alerts them to add items to the Manage List page, and includes a button to navigate them there. We installed the Material UI dependency and used their popup model, and we leveraged conditional rendering to determine when the popup model shows up.

## Related Issue



Closes #8 

## Acceptance Criteria


## Type of Changes

feature

## Updates

### After

![Screenshot 2024-09-03 at 10 47 55 AM](https://github.com/user-attachments/assets/7bb04ba5-2db1-453c-9f9e-22faa9ae368f)

## Testing Steps / QA Criteria

- use npm i to install MUI
- Create a new shopping list, that should be empty
- When redirected to the list page, a popup window should open
- Button on pop up window sends to you manage list
- If list is still empty, pop up window should still display until items are added
- Should be able to click out of pop up window
